### DISCUSSION TITLE
Adonis V5 API server: Post request latency

### DISCUSSION THREAD

**Discussion by Nikolastual** [adonisjs/core#1718] [Help]
I am using adonis 5 api server to use it as a backend in my react app.
I also use Axios for my GET, POST, UPDATE requests ...
I have a problem with my POST request to log in users.
I have a latency of 34,865ms when I do a post request to adonis.
When I connect to the adonis proxy with the url, I have latency too.
Adonis is launched with nginx

However everything works well, the connection to the member's area works...

Do you know where the problem can come from, how to debug it

<img width="1503" alt="Capture d’écran 2020-10-06 à 22 36 09" src="https://user-images.githubusercontent.com/43614012/95259739-f14c7a80-0827-11eb-8532-e57b4a83c620.png">

Thanks !

**Comment by thetutlage** [comment]
There can be things for the delay. First step is to kind if it is network latency or the server computation is slow.

Simply hit the API on localhost, if it is quick, that means server computation is not slow and it is the network latency, which can be coz of many reasons and one of them is the distance between the user and the server
